src/smolagents/monitoring.py

```diff
@@ -84,7 +86,7 @@ class LogLevel(IntEnum):
 class AgentLogger:
     def __init__(self, level: LogLevel = LogLevel.INFO):
         self.level = level
-        self.console = Console()
+        self.console = Console(width=100 if in_notebook() else None)
```

Comment:
What about using `force_jupyter`?
```suggestion
        self.console = Console(force_jupyter=True if in_notebook() else None)
```
This way_
- jupyter defaults are: (115, 100)
-  users can customize these values by setting the env variables `JUPYTER_COLUMNS` and `JUPYTER_LINES`

Reply:
Actually rich has a built in function _is_jupyter that provides this switch, so let's remove this change. I think the broken console was not in jupyter but in terminal, cannot find where, so I'll open a dedicated issue when the problem happens again.

Reply:
You can reproduce with this code (run from a jupyter notebook):
```python
!python -c "from rich.console import Console; console = Console(); print(console.size)"
```

Reply:
Output:
```
ConsoleDimensions(width=80, height=24)
```

Reply:
I get the same output as you: `ConsoleDimensions(width=80, height=24)`

But the problem I had was having rich cells too wide: a width value of 80, not being very large, probably is not an issue most of the time.